warehouse/accounts/views.py

```diff
@@ -344,22 +344,7 @@ def login(request, redirect_field_name=REDIRECT_FIELD_NAME, _form_class=LoginFor
                 # either where they were trying to go originally, or to the default
                 # view.
                 resp = HTTPSeeOther(redirect_to, headers=dict(headers))
-
-                # We'll use this cookie so that client side javascript can
-                # Determine the actual user ID (not username, user ID). This is
-                # *not* a security sensitive context and it *MUST* not be used
-                # where security matters.
-                #
-                # We'll also hash this value just to avoid leaking the actual User
-                # IDs here, even though it really shouldn't matter.
-                resp.set_cookie(
-                    USER_ID_INSECURE_COOKIE,
-                    hashlib.blake2b(
-                        str(userid).encode("ascii"), person=b"warehouse.userid"
-                    )
-                    .hexdigest()
-                    .lower(),
-                )
+                _set_userid_cookie(resp, userid)
```

Comment:
```suggestion
                _set_userid_insecure_cookie(resp, userid)
```


Since there are two cookies, I think it's probably best that this one is shown to be insecure in the call